### PR TITLE
Add parent/child.envFrom

### PR DESCRIPTION
**Pull request by rickard-von-essen**
This allows adding environment variables from `Secrets` and `ConfigMaps`
created outside of this Chart.

E.g. to read _Netdata Cloud claim token_ from an existing secret `netdata` set this to:

```yaml
parent:
  envFrom:
    - secretRef:
        name: netdata
```

And create it with:
```shell
$ kubectl create secret generic netdata --from-literal="NETDATA_CLAIM_TOKEN=<token>"
```

Also ensure that `parent.claim.token` is empty.

### CODE REVIEW

File: charts/netdata/values.yaml [hunk 1]
```diff
@@ -114,6 +114,13 @@ parent:
     ## To disable anonymous statistics:
     # DO_NOT_TRACK: 1
 
+  envFrom: []
+    ## E.g. to read Netdata Cloud claim token from an existing secret "netdata" set this to:
+    # - secretRef:
+    #     name: netdata
+    # And create it with: kubectl create secret generic netdata --from-literal="NETDATA_CLAIM_TOKEN=<token>"
+    # Also ensure that claim.token is empty
```
Comment (ilyam8) [inline charts/netdata/values.yaml:122]:
```suggestion
    ## And create it with: kubectl create secret generic netdata --from-literal="NETDATA_CLAIM_TOKEN=<token>"
    ## Also ensure that claim.token is empty
```

We use double `#` for comments

File: charts/netdata/values.yaml [hunk 2]
```diff
@@ -309,6 +316,13 @@ child:
...
```
Comment (ilyam8) [inline charts/netdata/values.yaml:324]:
```suggestion
    ## And create it with: kubectl create secret generic netdata --from-literal="NETDATA_CLAIM_TOKEN=<token>"
    ## Also ensure that claim.token is empty
```

**Review by ilyam8 — APPROVED** [review]

**Comment by ilyam8** [comment]
Thanks for the contribution, @rickard-von-essen 👍

**Review by M4itee — APPROVED** [review]
Thanks for that :)

**Pull request merged**